eZ/Publish/Core/SignalSlot/SignalDispatcher/SignalDispatcherFactory.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\SignalSlot\SignalDispatcher;
```

Comment:
This should be rather placed in `eZ\Publish\Core\Base\Container\ApiLoader`, maybe in a separate dir `SignalSlot`.